### DISCUSSION TITLE
API to find/filter courses having certificates

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Knowlg/Community#194] [Q&A]
Hello team
Hope you are doing well

Background -
We have updated few screens for sunbird-Ed and trying to add few more details

Issue Details -
Is there any API available which can tell me weather course or its batch have certificate?. I wanted to find total count of certificates achieved in total number of courses having certificates from my enrolled list

@komalm, @manishakapadnis, @AmiableAnil, @krgauraw ,@pallakartheekreddy

**Comment by pallakartheekreddy** [comment]
@paritshivani Above functionality is part of Lern building block. Can you create this thread in the below discussion forum.
https://github.com/orgs/Sunbird-Lern/discussions
